test/cases/chunks/named-chunks/index.js

```diff
@@ -33,3 +33,39 @@ it("should handle empty named chunks", function(done) {
 		sync = false;
 	});
 });
+
+it("should handle named chunks when there is an error callback", function(done) {
```

Comment:
This test interferes with the previous test, because it uses the same modules and chunk names. Use different chunk names and querystrings for the modules.

And use tabs instead of spaces.